tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/values_block.html

```diff
@@ -0,0 +1,28 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+<div class="values-header grid__values-header">
```

Comment:
Layout dictated that this was positioned differently in the overall page grid than the content below.